Snakefile

```diff
@@ -153,21 +155,113 @@ rule iq_tree:
         config["poppipe_location"] + "/scripts/run_iqtree.py"
 
+rule gubbins:
+    input:
+        alignment="output/strains/{strain}/align_variants.aln",
+        start_tree="output/strains/{strain}/besttree.nwk"
+    output:
+        final_tree="output/strains/{strain}/gubbins.final_tree.tre",
+    group:
+        "gubbins"
+    # log:
+    #     "logs/gubbins_{strain}.log"
+    params:
+        prefix=config['gubbins']['prefix'],
+        tree_builder=config['gubbins']['tree-builder'],
+        min_snp=config['gubbins']['min-snps'],
+        min_window=config['gubbins']['min-window-size'],
+        max_window=config['gubbins']['max-window-size'],
+        iterations=config['gubbins']['iterations'],
+        directory=config['gubbins']['dir'],
+        aln="align_variants.aln",
+        btree="besttree.nwk"
+
+    conda:
+        config["poppipe_location"] + "/envs/gubbins.yml"
+    threads:
+        4
+    shell:
+        "pushd {params.directory} &&\
+        run_gubbins.py {params.aln} --prefix {params.prefix} --starting-tree {params.btree} --threads {threads} \
+        && popd"
+
+rule bactdating:
+    input:
+        final_tree="output/strains/{strain}/gubbins.final_tree.tre",
+        metadata=config["transmission_metadata"]
+    output:
+        rds="output/strains/{strain}/bactdate_data.rds",
+        sorted="output/strains/{strain}/sorted.rds"
+    group:
+        "outbreak"
+    log:
+        "logs/bactdating_{strain}.log"
+    conda:
+        config["poppipe_location"] + "/envs/transphylo.yml"
+    threads:
+        4
+    shell:
+        "Rscript --vanilla scripts/run_bactDating.R output/strains/{wildcards.strain}/gubbins {input.metadata}  {output.sorted} {output.rds} > {log}"
+
+rule transmission:
+    input:
+        expand("output/strains/{strain}/transphylo_results.rds", strain=included_strain_ids)
+    output:
+        "done.txt"
+    shell:
+        "touch done.txt"
+
+
+rule transphylo:
+    input:
+        rds="output/strains/{strain}/bactdate_data.rds",
+        sorted="output/strains/{strain}/sorted.rds"
+    output:
+        rds="output/strains/{strain}/transphylo_results.rds"
+    group:
+        "outbreak"
+    params:
+        w_shape=config['transphylo']['w_shape'],
+        w_scale=config['transphylo']['w_scale'],
+        mcmcIterations=config['transphylo']['mcmcIterations'],
+        startNeg=config['transphylo']['startNeg'],
+        startOff_r=config['transphylo']['startOff_r'],
+        startOff_p=config['transphylo']['startOff_p'],
+        startPi=config['transphylo']['startPi'],
+        optiStart=config['transphylo']['optiStart'],
+        dateT=config['transphylo']['dateT'],
+        gubbins="output/strains/{strain}/gubbins"
+    log:
+        "logs/transphylo_{strain}.log"
+    conda:
+        config["poppipe_location"] + "/envs/transphylo.yml"
+    threads:
+        4
+    shell:
+        "Rscript --vanilla scripts/run_transPhylo.R --rds {input.rds} --sorted {input.sorted} --output {output}\
```

Comment:
Similarly here, if no save and no restore should be added

Reply:
same as above